internal/biz/model/reporter_representation.go

```diff
@@ -12,35 +12,24 @@ import (
 type ReporterRepresentation struct {
 	Representation
 
-	LocalResourceID    string  `gorm:"size:128;column:local_resource_id;index:reporter_rep_unique_idx,unique"`
```

Comment:
Keeping parity with CommonRepresentation, ReporterRepresentation will only store reporter attributes that need to be versioned. This is most similar to the ResourceHistory table in the legacy model. In addition to the attributes, it manages history fields across versions and generations. 

We also added a ReporterResourceID that is a foreign key reference to the surrogate key ResourceId in ReporterResource. We needed to add this surrogate key to enforce referential integrity since doing the same with a composite key (local_resource_id, reporter_type, resource_type, reporter_instance_id) is error prone and not the recommended option.

Reply:
All the fields removed from ReporterRepresentation have now moved to ReporterResource as an optimization